packages/react-dom/rollup.config.js

```diff
@@ -0,0 +1,26 @@
+import fs from "fs";
+import plugin from "@eik/rollup-plugin";
+import { createRequire } from "module";
+
+const { resolve } = createRequire(import.meta.url);
+const reactPkg = new URL("../react/package.json", import.meta.url);
+const reactDomPkg = new URL("./package.json", import.meta.url);
+
+const { name, main } = JSON.parse(fs.readFileSync(reactPkg.pathname, "utf8"));
+const { version } = JSON.parse(fs.readFileSync(reactDomPkg.pathname, "utf8"));
+
+const importMap = {
+  imports: {
+    react: `https://assets.finn.no/npm/${name}/${version}/${main}`,
```

Comment:
Dynamically build the React URL using the package.json of the `react` folder for both the `name` and filename while fetching the `version` field from this package (react-dom) to ensure that there is a hard dependency on the correct specific version of React.